tests/mechanic/telemetry_test.py

```diff
@@ -254,9 +255,8 @@ def test_raises_exception_on_transport_error(self):
                                      "cluster \[remote\]"):
             telemetry.CcrStatsRecorder(cluster_name="remote", client=client, metrics_store=metrics_store, sample_interval=1).record()
 
-
-    @mock.patch("esrally.metrics.EsMetricsStore.put_count_cluster_level")
-    def test_stores_default_ccr_stats(self, metrics_store_put_count):
+    @mock.patch("esrally.metrics.EsMetricsStore.put_doc")
+    def test_stores_default_ccr_stats(self, metrics_store_put_doc):
         total_fetch_time_millis = random.randint(0,9999999)
```

Comment:
nit: missing space after comma (this applies not only to this but all such calls in the test)